src/test/rpc/LedgerData_test.cpp

```diff
@@ -281,6 +282,180 @@ class LedgerData_test : public beast::unit_test::suite
         }
     }
 
+    /// @brief ledger_header RPC requests as a way to drive
+    /// input options to lookupLedger. The point of this test is
+    /// coverage for lookupLedger, not so much the ledger_header
+    /// RPC request.
```

Comment:
`ledger_header` is "deprecated and may be removed without further notice"
https://ripple.com/build/rippled-apis/#list-of-admin-commands
What do you think about moving this test to LedgerRPC_test.cpp and using the `ledger` command instead? I _think_ it would give us the same coverage in `lookupLedger`.